src/IconButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+import Icon from './Icon'
+import Button from './Button'
+import PropTypes from 'prop-types'
+
+const StyledButton = styled(Button)`
```

Comment:
I think it's generally better to give components like this more specific names, like *UnstyledButton* or *TransparentButton* – all of our buttons are "styled"